templates/zabbix-agent-aix.init.erb

```diff
@@ -0,0 +1,40 @@
+#!/bin/ksh
```

Comment:
thanks for the PR. can you please add a statement to this file that it's deployed via puppet and convert it to epp? We require each template to be epp instead of erb since some time.